doc/build-helpers/fetchers.chapter.md

```diff
@@ -755,25 +755,42 @@ Used with Subversion. Expects `url` to a Subversion directory, `rev`, and `hash`
 
 Used with Git. Expects `url` to a Git repo, `rev`, and `hash`. `rev` in this case can be full the git commit id (SHA1 hash) or a tag name like `refs/tags/v1.0`.
 
-Additionally, the following optional arguments can be given: `fetchSubmodules = true` makes `fetchgit` also fetch the submodules of a repository. If `deepClone` is set to true, the entire repository is cloned as opposing to just creating a shallow clone. `deepClone = true` also implies `leaveDotGit = true` which means that the `.git` directory of the clone won't be removed after checkout.
-
-If only parts of the repository are needed, `sparseCheckout` can be used. This will prevent git from fetching unnecessary blobs from server, see [git sparse-checkout](https://git-scm.com/docs/git-sparse-checkout) for more information:
-
-```nix
-{ stdenv, fetchgit }:
-
-stdenv.mkDerivation {
-  name = "hello";
-  src = fetchgit {
-    url = "https://...";
-    sparseCheckout = [
-      "directory/to/be/included"
-      "another/directory"
-    ];
-    hash = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
-  };
-}
-```
+Additionally, the following optional arguments can be given:
+
+- `fetchSubmodules`: Whether to also fetch the submodules of a repository.
+- `fetchLFS`: Whether to fetch LFS objects.
+- `postFetch`: Shell code executed after the file has been fetched successfully.
+  This can do things like check or transform the file.
+- `leaveDotGit`: Whether the `.git` directory of the clone should *not* be removed after checkout.
+  Be warned though that the git repository format is not stable and this flag is therefore not suitable for actual use by itself.
+  Only use this for testing purposes or in conjunction with removing the `.git` directory in `postFetch`.
```

Comment:
```suggestion
- `leaveDotGit`: Whether the `.git` directory of the clone should *not* be removed after checkout.

  Be warned though that the git repository format is not stable and this flag is therefore not suitable for actual use by itself.
  Only use this for testing purposes or in conjunction with removing the `.git` directory in `postFetch`.
```

This will ovoid a big blob in the list item, which makes it slightly more readable.

I'd generally recommend [using definition lists for function arguments](https://github.com/NixOS/nixpkgs/blob/master/doc/README.md#function-arguments) though.